spec/controllers/api/v0x0/tasks_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe Api::V0x0::TasksController, :type => :request do
+  it("Uses IndexMixin")   { expect(described_class.instance_method(:index).owner).to eq(Api::V0::Mixins::IndexMixin) }
+  it("Uses ShowMixin")    { expect(described_class.instance_method(:show).owner).to eq(Api::V0::Mixins::ShowMixin) }
+  it("Uses UpdateMixin")  { expect(described_class.instance_method(:update).owner).to eq(Api::V0::Mixins::UpdateMixin) }
+
+  let(:tenant) { Tenant.create! }
+
+  it "patch /tasks/:id updates a Task" do
+    task = Task.create!(:status => "started", :context => "context1", :tenant => tenant)
+
+    patch(api_v0x0_task_url(task.id), :params => {:status => "completed", :context => "context2"})
```

Comment:
I think the params need a `to_json` in order to be properly parsed from the body

Reply:
Ah good call, that was why I needed the `params_for_update` before, without the `to_json` the spec passed with `params_for_update`. Will fix.